### ISSUE TITLE
basestring not set

### ISSUE THREAD

**Issue by joefowler** [usnistgov/mass#175]
**[Original report](https://bitbucket.org/joe_fowler/mass/issues/175) by Jens Uhlig (Bitbucket: [erdzeichen](https://bitbucket.org/erdzeichen), GitHub: [erdzeichen](https://github.com/erdzeichen)).**

----------------------------------------

Following the demo or a manual minimal analysis.

pulled and compiled version with windows on Nov30th

```
run_0119_chan101.ljh 233334

chan 101 did not summarize because results were already preloaded
summarize_data 100.0% done, estimated ? left
summarize_data finished in 0.0 min
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-5-a9bfdd2d680b> in <module>
      3 data.summarize_data()
      4 ds=data.channel[101]
----> 5 ds.plot_summaries()

C:\ProgramData\Anaconda3\lib\site-packages\mass-0.7.0-py3.7-win-amd64.egg\mass\core\channel.py in plot_summaries(self, valid, downsample, log)
   1339 
   1340         # Convert "uncut" or "cut" to array of all good or all bad data
-> 1341         if isinstance(valid, basestring):
   1342             if "uncut" in valid.lower():
   1343                 valid = self.cuts.good()

NameError: name 'basestring' is not defined
```

**Comment by joefowler** [comment]
**Original comment by Jens Uhlig (Bitbucket: [erdzeichen](https://bitbucket.org/erdzeichen), GitHub: [erdzeichen](https://github.com/erdzeichen)).**

----------------------------------------

This is a python 2.7 vs 3 issue 

it needs to be a test for “`str”`

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Thanks, Jens. We’ll figure something out.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Remove py2-only use of "basestring"

Fixes #175.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Removing milestone: v0.7 (automated comment)
